### PR TITLE
add arm64 docker build support #379

### PR DESCRIPTION
**Pull request by shukla2009**
## Description

Support Multi-Platform Docker builds to allow KubeClarity Install on ARM-based platforms. #379

## Type of Change

[ ] Bug Fix
[x] New Feature
[ ] Breaking Change
[ ] Refactor
[ ] Documentation
[ ] Other (please describe)

## Checklist

- [x] I have read the [contributing guidelines](/CONTRIBUTING.md)
- [x] Existing issues have been referenced (where applicable)
- [x] I have verified this change is not present in other open pull requests
- [x] Functionality is documented
- [x] All code style checks pass
- [x] New code contribution is covered by automated tests
- [x] All new and existing tests pass

### CODE REVIEW

File: .github/workflows/reusable-build-and-push.yml [hunk 1]
```diff
@@ -34,6 +34,9 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
+      
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
```
[the remark this answers is not in the document]
Comment (shukla2009) [reply .github/workflows/reusable-build-and-push.yml:39]: Let me check on that

**Comment by shukla2009** [comment]
@FrimIdan please review

**Comment by tommy1199** [comment]
@shukla2009 Probably setting up QEMU is missing to make the GH workflow working again.

See https://github.com/marketplace/actions/docker-setup-buildx

`
     - # Add support for more platforms with QEMU (optional)
        # https://github.com/docker/setup-qemu-action
        name: Set up QEMU
        uses: docker/setup-qemu-action@v3
`

Otherwise the buildx builder does not support arm64 in later steps. QEMU can also be limited to arm64 only.

`
     - # Add support for more platforms with QEMU (optional)
        # https://github.com/docker/setup-qemu-action
        name: Set up QEMU
        uses: docker/setup-qemu-action@v3
        with:
          platforms: arm64
`

**Comment by chrisgacsal** [comment]
@tommy1199 The CI failed due to a network error. QEMU is not required anymore for multi-arch builds.

**Pull request closed without merging**